gitbootcamp.rst

```diff
@@ -159,28 +157,26 @@ Staging and Committing Files
 
 2. To stage the files to be included in your commit::
 
-      git add path/to/file1 path/to/file2 path/to/file3
+      git add <filename>
```

Comment:
I agree with the usage of ``<filename>`` instead of the current formatting, but I think it is also helpful to mention that multiple files can be added at once.
```suggestion
      git add <filename1> <filename2>
```

Reply:
Okay.